org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -18,7 +18,7 @@
 <repository location="http://download.eclipse.org/eclipse/updates/4.7/"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-<unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.3.0.v20170706-0836"/>
```

Comment:
Please sync with the origin master branch for this change to avoid merge conflict in future